src/server/pfs/s3/s3_test.go

```diff
@@ -0,0 +1,127 @@
+package main
```

Comment:
Package here should be `s3`, I think this is something that I probably did originally when I started this code. I'm actually a bit surprised go will compile a `main` package that doesn't have a `main` function in it.